src/DssDeploy.t.sol

```diff
@@ -709,6 +738,7 @@ contract DssDeployTest is DssDeployTestBase {
         assertEq(vat.wards(address(jug)), 1);
         assertEq(vat.wards(address(spotter)), 1);
         assertEq(vat.wards(address(end)), 1);
+        assertEq(vat.wards(address(esmAttack)), 1);
```

Comment:
Might want to assert that the bug ESM is _not_ authorized on the Vat...if it were, that would itself be a bug.

Reply:
Done!